### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by xalopp**

### CODE REVIEW

**Comment by calbrecht** [comment]
👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 If you contribute code, please make sure it conforms to the
 [PHPCS coding standard](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/phpcs.xml.dist)
 and that the unit tests still pass.
+When ever possible, add auto fixer for coding standard violations.
```
Comment (mmoll) [inline CONTRIBUTING.md]: "Whenever"

Comment (mmoll) [inline CONTRIBUTING.md]: ...add an auto fixer"

File: README.md [hunk 2]
```diff
@@ -145,12 +111,63 @@ Using [Composer](https://getcomposer.org/) is the preferred way.
...
 ## Contributing
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) for information.
 
-## Credit
-
 ## License
 
 This project is licensed under the MIT license.
+See [LICENSE](LICENSE) file for details.
```
Comment (mmoll) [inline README.md]: the.

File: README.md [hunk 3]
```diff
@@ -40,45 +34,17 @@ The MO4 Coding Standard is an extension of the [Symfony Coding Standard](http://
...
+## Requires
 
-```xml
-<rule ref="MO4.Formatting.AlphabeticalUseStatements">
-    <properties>
-        <property name="order" value="string-locale"/>
-    </properties>
-</rule>
-```
+* [PHP](http://php.net) 5.4 or any later
```
Comment (mmoll) [inline README.md]: ...version?
or "...or later"

File: README.md (hunk 2 above)
Comment (xalopp) [reply README.md]: hmm-kay

**Comment by mmoll** [comment]
merged, merci @xalopp!

**Pull request merged**